index.d.ts

```diff
@@ -0,0 +1,11 @@
+import { Redis, RedisOptions, ClusterOptions } from "ioredis";
+
+declare module 'egg' {
+  interface Application {
+    redis: Redis;
+  }
+
+  interface EggAppConfig {
+    redis: RedisOptions | ClusterOptions;
```

Comment:
It seems incorrect with this. see https://github.com/eggjs/egg-redis/blob/master/config/config.default.js#L3